roles/network_plugin/cilium/templates/cilium-ds.yml.j2

```diff
@@ -162,6 +162,9 @@ spec:
             - name: docker-socket
               mountPath: /var/run/docker.sock
               readOnly: true
+            - name: crio-socket
+              mountPath: /var/run/crio.sock
```

Comment:
what if crio is not used ?

Reply:
I think kubelet does not check for the hostpath (whether it exists or not) as passes it to container runtime. Both docker and crio has the default behaviour of creating a directory for the hostpath if it does not exist. So there would be a redundant hostpath mount, but it works fine with both docker/crio in my testing.

Reply:
can you suround with: 
```
{% if container_manager == 'crio' %}
(crio-mount)
{% else %}
(docker-mount)
{% endif %}
```